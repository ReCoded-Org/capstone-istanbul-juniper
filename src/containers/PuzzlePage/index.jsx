import React from "react";
import PuzzlePageContent from "./../../components/PuzzlePageContent/index";

const PuzzlePage = () => {
  return (
    <>
      <PuzzlePageContent />
    </>
  );
};

export default PuzzlePage;
