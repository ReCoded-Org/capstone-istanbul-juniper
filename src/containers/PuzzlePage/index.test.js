import React from "react";
import PuzzlePage from "./index.jsx";
import renderer from "react-test-renderer";

describe("PuzzlePage Component", () => {
  it("matches the tree", () => {
    const tree = renderer.create(<PuzzlePage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
