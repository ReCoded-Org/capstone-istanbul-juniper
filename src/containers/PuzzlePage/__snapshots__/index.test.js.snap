// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PuzzlePage Component matches the tree 1`] = `
<main
  className="ant-layout-content puzzlePage__content"
>
  <div
    className="puzzlePage__content__titles"
  >
    <h2
      className="puzzlePage__content__title"
    >
      games.puzzleGame.title
    </h2>
    <p
      className="puzzlePage__content__subtitle"
    >
      games.puzzleGame.subtitle
    </p>
  </div>
  <div
    className="ant-card ant-card-small gameContainer"
  >
    <div
      className="ant-card-body"
      style={Object {}}
    >
      <div
        className="ant-col gameContainer__listOfImages"
        style={Object {}}
      >
        <div
          className="ant-row gameContainer__listOfImages--imgContainer"
          onClick={[Function]}
          style={Object {}}
        >
          <img
            alt="Click to choose your puzzle"
            className="gameContainer__listOfImages--img"
            src="habitat.jpg"
          />
        </div>
        <div
          className="ant-row gameContainer__listOfImages--imgContainer"
          onClick={[Function]}
          style={Object {}}
        >
          <img
            alt="Click to choose your puzzle"
            className="gameContainer__listOfImages--img"
            src="conservation.jpg"
          />
        </div>
        <div
          className="ant-row gameContainer__listOfImages--imgContainer"
          onClick={[Function]}
          style={Object {}}
        >
          <img
            alt="Click to choose your puzzle"
            className="gameContainer__listOfImages--img"
            src="holdingPlant.jpg"
          />
        </div>
        <div
          className="ant-row gameContainer__listOfImages--imgContainer"
          onClick={[Function]}
          style={Object {}}
        >
          <img
            alt="Click to choose your puzzle"
            className="gameContainer__listOfImages--img"
            src="planting.jpg"
          />
        </div>
      </div>
      <div
        className="ant-col gameContainer__puzzleGrid"
        style={Object {}}
      >
        <div
          className="ant-col puzzleGrid"
          style={Object {}}
        />
      </div>
    </div>
  </div>
</main>
`;
